schemas/src/shared/AuthCardButton.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/AdaptiveCards/6f39aedce45864ae1067ed44a5551dc973790bb5/source/nodejs/typed-schema/schema/lib/Type.json",
+    "description": "Defines a button as displayed when prompting a user to authenticate. This maps to the cardAction type defined by the Bot Framework (https://github.com/microsoft/botframework-obi/blob/jeffders/adaptiveCards2/protocols/botframework-activity/botframework-activity.md#card-action).",
+    "version": "1.4",
+    "properties": {
+        "type": {
+            "type": "string",
```

Comment:
Seems like there are other possible "types", not just "signin", that we may use in the future? If so, is AuthCardButton too specific of a name for this schema?

Reply:
It maps directly to the type names defined in the existing Bot protocol. This is a case of reuse of what already existed before Universal Actions, to maximize code reuse.